tests/signatures.rs

```diff
@@ -56,17 +56,17 @@ fn ed25519_key_and_ed25519_detects_bad_signature() {
 fn ed25519_key_rejected_by_other_algorithms() {
     let ee = include_bytes!("signatures/ed25519.ee.der");
     for algorithm in &[
-        webpki::ECDSA_P256_SHA256,
-        webpki::ECDSA_P256_SHA384,
-        webpki::ECDSA_P384_SHA256,
-        webpki::ECDSA_P384_SHA384,
-        webpki::RSA_PKCS1_2048_8192_SHA256,
-        webpki::RSA_PKCS1_2048_8192_SHA384,
-        webpki::RSA_PKCS1_2048_8192_SHA512,
-        webpki::RSA_PKCS1_3072_8192_SHA384,
-        webpki::RSA_PSS_2048_8192_SHA256_LEGACY_KEY,
-        webpki::RSA_PSS_2048_8192_SHA384_LEGACY_KEY,
-        webpki::RSA_PSS_2048_8192_SHA512_LEGACY_KEY,
+        webpki::ring::ECDSA_P256_SHA256,
```

Comment:
Could/should these (and the other long lists in tests) use `ALL_`?

Reply:
some of these lists are things like "all the algorithms except RSA_PSS_2048_8192_SHA256_LEGACY_KEY" which could be addressed by filtering the ALL_ list and removing the desired one? probably not going to do that in this PR, but will keep it in mind for a later refactor